frontend/src/e2e-test/pages/admin/application-workbench-page.ts

```diff
@@ -54,6 +54,13 @@ export class ApplicationWorkbenchPage {
   #applicationNoteContent = this.page.find(
     '[data-qa="application-note-content"]'
   )
+
+  #applicationPlacementProposalStatusIndicator = this.page.find(
+    '[data-qa="placement-proposal-status"]'
+  )
+  #applicationPlacementProposalStatusTooltip = this.page.find(
+    '[data-qa="placement-proposal-status-tooltip"]'
+  )
```

Comment:
Perhaps worth another PR, but nowadays we have `page.findByDataQa()` helper that removes the need for the `[data-qa="` boilerplate in selectors.